core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/scheduling/config/ResourceConfig.scala

```diff
@@ -26,4 +26,5 @@ case class ResourceConfig(
     operatorConfigs: Map[PhysicalOpIdentity, OperatorConfig] = Map.empty,
     linkConfigs: Map[PhysicalLink, LinkConfig] = Map.empty,
     portConfigs: Map[GlobalPortIdentity, PortConfig] = Map.empty
+    // TODO: Separate into two types
```

Comment:
move it to before the port Config and make the subject (PortConfig) clear, so that we know the comment is about the PortConfig. otherwise we will soon forget what do you mean.